src/App.js

```diff
@@ -29,20 +30,15 @@ const alex = {
 const App = () => (
   <div className="App">
     <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
+      <Person {...misha} />
     </section>
 
     <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
+      <Person {...olya} />
     </section>
 
     <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
+      <Person {...alex} />
```

Comment:
Please, read this:
http://mateacademy.slack.com/archives/C03C2C13KRA/p1657005525754959